keras_contrib/callbacks/dead_relu_detector.py

```diff
@@ -25,7 +25,11 @@ def __init__(self, x_train, verbose=False):
 
     @staticmethod
     def is_relu_layer(layer):
-        return isinstance(layer, Dense) and layer.get_config()['activation'] == 'relu'
+        # Should work for all layers with relu activation. Tested for Dense and Conv2D
+        if 'activation' in layer.get_config():
+            return layer.get_config()['activation'] == 'relu'
+        else:
+            return False
```

Comment:
@infelane 
You can simplify this to
`return 'activation' in layer.get_config() and layer.get_config()['activation'] == 'relu'`

Reply:
Indeed! How do I just recommit-repush-repull?